packs/agents-of-edgewatch-bestiary/hestriviniaas.json

```
@@ -2730,7 +2730,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>A creature struck by Hestriviniaas' jaws or claw Strike must succeed at a @Check[type:fortitude|dc:45|traits:polmorph] save or be subject to a @RollTable[cBHMpmeIhoBdUyQT]{Warpwave}.</p>"
+                    "value": "<p>A creature struck by Hestriviniaas' jaws or claw Strike must succeed at a @Check[type:fortitude|dc:45|traits:polymorph] save or be subject to a @RollTable[cBHMpmeIhoBdUyQT]{Warpwave}.</p>"
```

```suggestion
                    "value": "<p>A creature struck by Hestriviniaas' jaws or claw Strike must succeed at a @Check[type:fortitude|dc:45] save or be subject to a @UUID[Compendium.pf2e.rollable-tables.RollTable.Warpwaves].</p>"
```

It doesn't actually have that trait.

Also, that roll table link is broken, there's a handful of others with the same issue, all from AoE.